gamecenter/api/helpers/meta.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""API helpers for META info."""
+from flask import current_app
+
+from gamecenter.api.models import Score
+
+
+def get_meta_from_args(args):
+    """Construct the meta dictionary from the args"""
+    meta = {}
+    uri = current_app.config['HOST_URI']
+    meta['total'] = Score.query.count()
```

Comment:
yes, this is a placeholder, I just thought that if we were both going to use it, better to have a placeholder so we both work off of the common interface